.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,89 @@
+name: CDN Code Deploy 
+description: Assets deployment to CDN
+
+# This action allows you to deploy the passed asset to the CDN.
+# The asset must be the directory that contains all the files you want to sync.
+
+inputs:
+  storage_account_name:
+    description: Storage account name.
+    type: string
+    required: true
+  blob_container_name:
+    description: Blob container name.
+    type: string
+    required: true
+  resource_group_name:
+    description: CDN resource group name.
+    type: string
+    required: true
+  profile_name:
+    description: CDN Profile name.
+    type: string
+    required: true
+  endpoint_name:
+    description: CDN endpoint name.
+    type: string
+    required: true
+  selective_purge_paths:
+    description: |
+      List of relative paths for files to include in the purge. Leave empty to purge all files.
+      Paths should be relative to the synchronized directory (sync_dir_name).
+      For example, if syncing ./dist, to include ./dist/path/file1 and ./dist/path/file2, specify "./path/file1 ./path/file2."
+      To recursively purge an entire directory, use the pattern "./path/dir/*".
+
+      The format should follow the parameters as described in the official documentation: 
+      https://learn.microsoft.com/en-us/cli/azure/cdn/endpoint?view=azure-cli-latest#az-cdn-endpoint-purge-required-parameters
+    type: string
+    required: false
+  sync_dir_name:
+    description: The asset that is the name of the directory containing the files to be synchronized (Default is 'dist')
+    type: string
+    required: false
+  working_directory:
+    description: The path of the directory containing the folder to be synchronized (Default is '.')
+    type: string
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+    ## Sync Storage
+    # will be synced the to the working_directory/sync_dir_name defined (Default will be './dist')
+    - name: Sync Storage
+      shell: bash
+      run: |
+        az storage blob sync \
+          -c ${{ inputs.blob_container_name }} \
```

Comment:
I suggest to use the entire command name in scripts, in order to improve readability

Reply:
shouldn't inputs be escaped?